.travis.yml

```diff
@@ -43,6 +43,8 @@ install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - npm install phantomjs-prebuilt@2.1.14
+  - export PATH=${TRAVIS_BUILD_DIR}/node_modules/phantomjs-prebuilt/lib/phantom/bin:${PATH}
```

Comment:
These changes can be reverted - it is [now in MQT](https://github.com/OCA/maintainer-quality-tools/blob/5b4b67e757611ac393aaf4434195ca5aa0cd4502/travis/travis_install_nightly#L35)